app/views/schools/_form.html.erb

```diff
@@ -1,9 +1,41 @@
-<div class="container">
+<table class="form-table">
```

Comment:
Same ^

Reply:
I changed all the tables to be div with lists and I adapted the scss. All in sha: 3361eb6